tests/monitoring/monitoring_service/test_handlers.py

```diff
@@ -169,7 +169,7 @@ def test_channel_opened_event_handler_adds_channel(context: Context):
 
 def test_channel_closed_event_handler_closes_existing_channel(context: Context):
     context = setup_state_with_open_channel(context)
-    context.web3.eth.blockNumber = BlockNumber(60)
+    context.web3.eth.blockNumber = BlockNumber(60)  # type: ignore
```

Comment:
What is the typing problem, here?

Reply:
error: Property "blockNumber" defined in "Eth" is read-only